docker/configs/settings_local.py

```diff
@@ -40,7 +40,7 @@
 #    'ietf.context_processors.sql_debug',
 # ]
 
-DOCUMENT_PATH_PATTERN = '/assets/ietf-ftp/{doc.type_id}/'
+DOCUMENT_PATH_PATTERN = '/assets/ietfdata/doc/{doc.type_id}/'
```

Comment:
Should we move the assets around on the dev container before merging this, or deal with it later? (Earlier you said there'd be other fallout -- just confirming whether this is intentional or was forgotten)